File: LeetCode/CSharp/3110-ScoreOfAString.cs
using LeetCode.Shared;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace LeetCode.CSharp
{
    internal class Solution3110 : IRunProgram
    {
        public void Run()
        {
            ScoreOfString("hello").Print();
            ScoreOfString("zaz").Print();
        }

        public int ScoreOfString(string s)
        {
            int score = 0;

            for (int i = 0; i < s.Length; i++)
            {
                if (i + 1 >= s.Length)
                {
                    break;
                }

                score += Math.Abs(s[i] - s[i + 1]);
            }

            return score;
        }
    }
}

File: Java/3110-score-of-a-string.java
class Solution {
    public int scoreOfString(String s) {
        int output = 0;

        for (int i = 0; i < s.length() - 1; i++) {
            output += Math.abs(s.charAt(i) - s.charAt(i + 1));
        }

        return output;
    }
}